### ISSUE TITLE
250+ Solitaire Collection app - ads

### ISSUE THREAD

**Issue by evgeniy-ADG** [AdguardTeam/AdguardFilters#4232]
https://play.google.com/store/apps/details?id=com.anoshenko.android.solitaires&hl=en&rdid=com.anoshenko.android.solitaires

On every start, the game opens a browser with the following tab: alxanosoft.com/rk/pintel/index.html

<details>

![screenshot_2017-01-26-07-13-55](https://cloud.githubusercontent.com/assets/15856304/22403235/d7108758-e619-11e6-9ed6-df4ff37c3381.png)

</details>

***Steps to reproduce the problem***:

1. Download the game
2. Launch the game
3. See the issue

[//]: # (Describe, how to reproduce the issue)
[//]: # (Перечислите шаги для воспроизведения проблемы)

***System configuration***

Information | value
--- | ---
Operating system: | Android
Browser: | Chrome. Any.
Adguard version: | 2.8, the current release
Adguard mode (Android): | VPN
Filtering quality (Android): | High-quality
HTTPS filtering (Android): |  On, Default
Adguard DNS: | Default
Adguard filters: | [ 2, 3, 4, 14, 11, 15 ]

[//]: # (***Filters list - comma separated list, or filter names outside the table)
[//]: # (***Список фильтров - названия через запятую, или список вне таблицы)

Ticket ID(if exists): 1491360

[//]: # (This template is for problem reports, for other type of reports edit it accordingly)
[//]: # (If this is a crash report, include the crashlog with https://gist.github.com/)

**Comment by Alex-302** [comment]
fixed

**Comment by evgeniy-ADG** [comment]
>During game play all is fine. I am not receiving any pop-ups. However, when
you win the game there is an immediate pop-up direct to the browser
(Chrome). I've attached a screenshot of the web page that opens. It appears
to be the same as the previous pop up which appeared as an in-game window
as soon as I opened the game to play. I've attached that screenshot as
well. I've also sent the feedback via the app. Thanks.

HTTP requests log, the time of the occurrence 13-15
[debug (2).txt](https://github.com/AdguardTeam/AdguardFilters/files/752235/debug.2.txt)
